drc_cmis/client.py

```diff
@@ -212,7 +212,9 @@ def update_gebruiksrechten(self, drc_uuid: str, data: dict) -> Gebruiksrechten:
 
         return gebruiksrechten.update_properties(diff_properties)
 
-    def create_oio(self, data: dict) -> ObjectInformatieObject:
+    def create_oio(
+        self, oio_data: dict, zaak_data: dict = None, zaaktype_data: dict = None
```

Comment:
the default values of `None` make it seem like these fields are optional, but I don't see a fallback where the data is retrieved if not provided upfront. I can see that these parameters are not available (and not relevant) if we're relating a document and a besluit, but if we're dealing with a zaak I'd make my code a bit more defensive:

```py
if oio_data["object_type"] == "zaak" and (not zaak_data or not zaaktype_data):
    raise ValueError("You must provide 'zaak_data' and 'zaaktype_data' when relating documents to zaken")
```